src/main/java/screens/CreateEventScreen.java

```diff
@@ -112,6 +116,10 @@ private void addTimeFields(JPanel panel, GridBagConstraints constraints) {
         constraints.gridwidth = 1; // component spans 1 column
         constraints.gridy = 3; // row 3
         constraints.gridx = 1; // col 1
+        addPanel(panel, constraints, startTimeLabel, endTimeLabel, startTime, endTime);
```

Comment:
I disagree with this refactoring. What is added to the panels for each screen should remain specific to each screen, and making a shared helper method for adding the time fields to the panel between the create and modify event screens would cause issues should we decide to rearrange the layout of one screen but not the other. We should ignore this warning and document why it was ignored, which is approximately what I have said in this comment.